docs/issues.md

```diff
@@ -1,9 +1,9 @@
 # Filing An Issue
-The Openshift Aggregated Logging integration is a multicomponent technology stack whose individual 
-parts must all work together to gather and archive logs.  It is important to include as much information 
-as possible when issues arise in order to properly debug and diagnose.  Run 
-**ADD SCRIPT OR UTILITY HERE** which will gather the desired information.  Alternatively, you can gather 
-them manually. 
+The Openshift Aggregated Logging integration is a multicomponent technology stack whose individual
+parts must all work together to gather and archive logs.  It is important to include as much information
+as possible when issues arise in order to properly debug and diagnose.  Run
+[logging gathering script](../hack/dump/logging-dump.sh) which will gather the desired information.  Alternatively, you can gather
```

Comment:
@richm I agree but as it had the detailed readme file I thought it should lay on a different folder or else we would need to adapt the readme to explain the other scripts